Formula/openlitespeed.rb

```diff
@@ -11,19 +11,83 @@ class Openlitespeed < Formula
     sha256 "aacafddf229c07c76481f3fc86b3295ca06b41e14555b0e8a97050098701a1d2" => :mavericks
   end
 
-  option "with-debug", "Compile with support for debug log"
-  option "with-spdy", "Compile with support for SPDY module"
+  option "with-debug", "Build with support for debug log"
+  option "with-redis", "Enable redis for cache module"
+
+  option "without-http2", "Disable SPDY and http2"
 
   depends_on "pcre"
   depends_on "geoip"
   depends_on "openssl"
+  depends_on "expat"
+  depends_on "zlib"
 
   def install
-    args = ["--disable-dependency-tracking", "--prefix=#{prefix}"]
+    args = %W[
+      --disable-dependency-tracking
+      --prefix=#{prefix}
+      --with-pcre=#{Formula["pcre"].opt_prefix}
+      --with-openssl=#{Formula["openssl"].opt_prefix}
+      --with-zlib=#{Formula["zlib"].opt_prefix}
+      --with-expat=#{Formula["expat"].opt_prefix}
+      --with-expat-inc=#{Formula["expat"].opt_prefix}/include
+      --with-expat-lib=#{Formula["expat"].opt_prefix}/lib
+    ]
+
     args << "--enable-debug" if build.with? "debug"
-    args << "--enable-spdy" if build.with? "spdy"
-    args << "--with-openssl=#{Formula["openssl"].opt_prefix}"
+    args << "--enable-redis=yes" if build.with? "redis"
+    args << "--enable-http2=no" if build.without? "http2"
+
+    ENV["CPPFLAGS"] = "-I#{Formula["openssl"].opt_prefix}/include/openssl"
```

Comment:
according to this [testReport](http://bot.brew.sh/job/Homebrew%20Core%20Pull%20Requests/3758/testReport/), specifying `CPPFLAGS` seem to solve the `OpenSSL too low error`